src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -115,6 +115,8 @@ class DraftAdjudicationService(
     submittedAdjudicationHistoryRepository.save(
       SubmittedAdjudicationHistory(
         adjudicationNumber = reportedAdjudication.adjudicationNumber,
+        agencyId = reportedAdjudication.agencyId,
```

Comment:
We need to store these so we can search on them

Reply:
Cool - I guess as time progresses we might need to add more data